parkhangs-webapp/src/components/ParkFilterToolbar.js

```diff
@@ -295,6 +389,14 @@ class ParkFilterToolbar extends React.Component {
         }
     }
 
+    clearAllLocation = () => {
+        this.setState({distance: '', lat: '', lon: '', formattedPlaceString: ''})
+    }
+
+    clearLocation = () => {
+        this.setState({lat: '', lon: '', formattedPlaceString: ''})
+    }
+
     handleClearSearch = event => {
```

Comment:
Where is this function being called from?

Reply:
the AddressSearchbar.js file

Reply:
when the input box detects a delete and makes the address invalid